include/avif/avif.h

```diff
@@ -723,7 +723,9 @@ typedef struct avifImage
     avifImageMirror imir;
 
     // Metadata - set with avifImageSetMetadata*() before write, check .size>0 for existence after read
-    avifRWData exif;
+    avifRWData exif; // exif_payload chunk from the ExifDataBlock specified in ISO/IEC 23008-12:2022 Section A.2.1.
+                     // The value of the 4-byte exif_tiff_header_offset field, that is not part of this avifRWData
+                     // byte sequence, can be retrieved by calling avifGetExifTiffHeaderOffset(avifImage.exif).
```

Comment:
In our decoder, should we validate that the value of `exif_tiff_header_offset` is equal to the output of `avifGetExifTiffHeaderOffset()`?